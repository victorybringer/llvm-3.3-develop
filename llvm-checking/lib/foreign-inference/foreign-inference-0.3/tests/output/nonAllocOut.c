#include <stdlib.h>

int * g;

void outAlloc(int n, int** arr) {
  // This is not an alloc-out parameter because the result escapes
  // via a different assignment.
  *arr = malloc(n * sizeof(int));
  g = *arr;
}
