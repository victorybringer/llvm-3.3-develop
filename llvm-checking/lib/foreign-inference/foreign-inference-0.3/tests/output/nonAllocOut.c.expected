fromList [ ("@outAlloc", fromList [("%arr", PAOut)]) ]
